switchsdk/src/main/java/net/gini/switchsdk/camera/Camera1.java

```diff
@@ -129,6 +129,11 @@ public void surfaceChanged(final SurfaceHolder surfaceHolder, final int i, final
                     parameters.setFocusMode(FOCUS_MODE_CONTINUOUS_PICTURE);
                 }
 
+                final List<String> supportedFlashModes = parameters.getSupportedFlashModes();
```

Comment:
that's where the camera parameters for the resolution have to be set up, because they depend on the surface size. I want to setup the whole parameters at one place and camera and Android are mean. In the `surfaceCreated` method there is also an old comment that says something like: not here. But I agree it's long and I will refactor a little bit there.